central/graphql/resolvers/images.go

```diff
@@ -34,22 +34,27 @@ func registerImageWatchStatus(s string) string {
 func init() {
 	schema := getBuilder()
 	utils.Must(
+		schema.AddExtraResolvers("Image", []string{
+			"deployments(query: String, pagination: Pagination): [Deployment!]!",
+			"deploymentCount(query: String): Int!",
+			"topVuln(query: String): EmbeddedVulnerability",
+			"vulnerabilities(query: String, scopeQuery: String, pagination: Pagination): [ImageVulnerability]!",
+			"vulnerabilityCount(query: String): Int!",
+			"vulnerabilityCounter(query: String): VulnerabilityCounter!",
+			"vulns(query: String, scopeQuery: String, pagination: Pagination): [EmbeddedVulnerability]!",
+			"vulnCount(query: String): Int!",
```

Comment:
Since we have both RN, let's add a comment for the sub-resolvers we plan to deprecate in future. We do not want someone prefer the ones that will be deprecated.